### DISCUSSION TITLE
Stremio Streaming Service on Winutil?

### DISCUSSION THREAD

**Discussion by ToastMaster63** [ChrisTitusTech/winutil#1656] [Ideas]
Hi everyone,

I've got a cool idea that I think could make the Winutil project even better, let's add Stremio to the mix! Stremio is a fantastic streaming platform with tons of movies, TV shows, and channels all for free works for Windows, MacOS, Linux, and many more. overall what I'm saying it's an amazing streaming service without any fees and I would love to see it available for everyone. 

Here's the link for anyone interested: [https://www.stremio.com](url)

**Comment by REVENGE977** [comment]
It's already there. [I added it here](https://github.com/ChrisTitusTech/winutil/pull/1473).

![image](https://github.com/ChrisTitusTech/winutil/assets/32039975/dd819e92-520d-4de2-afc4-0e399415199a)

**Reply by ToastMaster63** [reply]
ohh nice for some reason I didn't see that at first. thanks
